### DISCUSSION TITLE
Axis: Support compose TextStyle/Typeface/Font

### DISCUSSION THREAD

**Discussion by zoltish** [patrykandpatrick/vico#309] [Ideas]
Its currently very awkward to integrate custom fonts into the axis labels when using compose. Id love to see this simplified, although I understand that its a complex thing to solve given the view based drawing. If you have any suggestions/workarounds, feel free to share them below as well. 

Edit: Perhaps this is not needed. I just needed to Google harder for a [solution](https://stackoverflow.com/questions/70800896/how-to-convert-textstyle-from-jetpack-compose-to-android-graphics-typeface).

**Comment by patrickmichalik** [comment]
Hello! I’m glad you’ve found a solution. It’s worth noting that the core logic isn’t, as such, view-based. When you create a chart using the `Chart` composable function, Vico doesn’t do anything related to views. The `core` module utilizes APIs from `android.graphics`. Generally, both the view system and Jetpack Compose use these APIs, but it tends to be the case that the view system does so more directly.
